checkstyle/spotbugs-exclude.xml

```diff
@@ -8,7 +8,7 @@
     <Match> <!-- No practical consequence and generate a lot of warnings -->
```

Comment:
Looks like both the files (include and exclude.xml) don't have the copyright headers. Could you please add?